docs/source/models/risk_correlation/sbp_ldlc_fpg_bmi/index.rst

```diff
@@ -0,0 +1,89 @@
+
+- :ref:`LDL-C risk exposure <2019_risk_exposure_ldl>`
+
+- :ref:`FPG risk exposure <2019_risk_exposure_fpg>`
+
+- :ref:`BMI risk exposure <2019_risk_bmi>`
+
+Vivarium Modeling Strategy
+----------------------------
+
+Correlation
+++++++++++++
+
+Correlation for all risk factors was derived from NHANES data. The dataset included 
+surveys between 1988 and 2018. The data was examined and correlation coefficients were calculated in
+the `R file here <https://github.com/ihmeuw/vivarium_research_nih_us_cvd/blob/main/correlation.R>`_. 
+
+We saw that there was a relationship between age and correlation coefficient for some 
+of the variables. This can be visualized in the graphs below. All graphs have age on the x-axis 
+and the correlation coefficient on the y-axis. 
+
+.. image:: correlation_graphs.png
+
+With our clinical expert, we used this information and decided that the correlation 
+would be included in the following way: 
+
+- For SBP and BMI, correlation would be age specific 
+- For SBP/LDL-C, SBP/FPG, LDL-C/BMI, and FPG/BMI the age effects were limited and thought to be primarily noise. Therefore they are included at an overall level, not age specific. 
+- LDL-C and FPG were not statistically significantly correlated for most ages and therefore correlation was set to 0 
+
+The correlation was then recalculated based on the above (age specific or not) 
+and the final values for correlation coefficients are stored in the csv file 
+here: /ihme/costeffectiveness/artifacts/vivarium_nih_us_cvd/raw_data/correlation.csv 
+
+The correlation is then used to create propensity values for all simulants. Therefore, the 
+correlation will continue to be the same even as simulants age and their risk exposure values 
+change. 
+
+This is accomplished using a multivariate normal distribution with a defined 
+correlation matrix matching our calculated correlation coefficients. From the 
+multivariate, a cdf function is used to convert the values into a uniform distirbution 
+between 1 and 0 with the same correlation between variables. 
+
+The calculations are done `in this notebook <https://github.com/ihmeuw/vivarium_research_nih_us_cvd/blob/main/Correlation_Testing.ipynb>`_. This code can be used to generate propensity 
+values for any number of simulants. 
+
+Causation
++++++++++
+
+We are not currently modeling a direct causal relationship between changes in 
+any risk exposure values. 
+
+Assumptions and Limitations
+++++++++++++++++++++++++++++++
+
+#. We assign a correlation at initialization that does not change over time. For SBP and BMI which we model to change over time, this means that as simulants age, they will retain their original correlation and increasly differ from the true correlation. 
+
+#. To calculate the correlation, we limited the population to only untreated individuals. However, as our simulants receive treatments their risk exposure values will change and likely make validation more challenging. 
+
```

Comment:
Have we examined how the treated correlation coefficients differ from the untreated? I'm mostly just curious, but we could use this to inform what to expect here and/or even use that data directly if needed

Reply:
I looked again at the untreated values. They are not very different - in general there is actually a bit higher correlation and fewer very low outliers in the older age groups. The outliers is probably since so many people are on treatment by the time they are 80+ that we don't get great results on untreated only. 

However, the bigger issue you pointed out is that we actually assign _treated_ exposure values to the population from GBD. Then afterwards, we pick people to be on treatment and create artificial _untreated_ exposures, and then reduce them with treatment to their original value obtained through GBD. [More information](https://vivarium-research.readthedocs.io/en/latest/models/concept_models/vivarium_us_cvd/concept_model.html#creation-of-untreated-sbp-values-on-initializaiton) 

Therefore, I actually think we should use the full NHANES population in finding the correlation coefficients. Conveniently, this is also how we decide how many folks are on treatment, so by definition it will match the model. Does that make sense to you? If so, I can update the values but I don't expect things to change dramatically.

Reply:
Yeah, that makes sense to me! Good to know it won't make too much of a difference, but yeah seems like it would be best to update then

Reply:
Thanks!!